coral-erm/coral
# Add library number to orders and dashboards

Test plan:
      - Apply the database update
      - Check that you can add/edit/remove a library number in orders
      - Check that you can group by library number in the dashboards

![librarynumber](https://user-images.githubusercontent.com/1679997/49520719-3017ef00-f8a4-11e8-89cb-fefff628c9cc.png)
![librarynumberstats](https://user-images.githubusercontent.com/1679997/49520721-31e1b280-f8a4-11e8-8313-32d83ed02fdb.png)
